.github/workflows/publish_pr_build.yml

```diff
@@ -71,21 +71,33 @@ jobs:
         run: |
           wget -O eve-arm64.zip "${{steps.find-pr-artifact-arm64.outputs.result}}"
 
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
```

Comment:
Do we really need to use external action here? It may affect security as it is not under our control.

Reply:
Agree with @giggsoff -- I really would like to avoid this.

Reply:
Ok, I will avoid using external actions.